tests/osprh960

```diff
@@ -0,0 +1,12 @@
+#type=AVC msg=audit(11/21/2023 16:15:41.602:56173) : avc:  denied  { read append } for  pid=194003 comm=virtlogd name=system.token dev="tmpfs" ino=2605 scontext=system_u:system_r:virtlogd_t:s0-s0:c0.c1023 tcontext=system_u:object_r:virt_var_run_t:s0 tclass=file
+type=AVC msg=audit(11/21/2023 18:43:16.798:56328) : avc:  denied  { getattr } for  pid=194510 comm=virtlogd path=/run/libvirt dev="tmpfs" ino=2396 scontext=system_u:system_r:virtlogd_t:s0-s0:c0.c1023 tcontext=system_u:object_r:container_file_t:s0 tclass=dir
+#type=AVC msg=audit(11/21/2023 18:44:45.879:56406) : avc:  denied  { open } for  pid=194763 comm=virtlogd path=/run/libvirt/common/system.token dev="tmpfs" ino=2605 scontext=system_u:system_r:virtlogd_t:s0-s0:c0.c1023 tcontext=system_u:object_r:virt_var_run_t:s0 tclass=file
+#type=AVC msg=audit(11/21/2023 18:44:45.879:56407) : avc:  denied  { lock } for  pid=194763 comm=virtlogd path=/run/libvirt/common/system.token dev="tmpfs" ino=2605 scontext=system_u:system_r:virtlogd_t:s0-s0:c0.c1023 tcontext=system_u:object_r:virt_var_run_t:s0 tclass=file
```

Comment:
Could we remove the commented out denials from the file altogether? From what I understand, we're confident these were due to the chcon and won't be necessary anyway.

Reply:
it really helps if i git add before i push :)